### DISCUSSION TITLE
Assistance Needed: Sigma Detection and Playbook Customization in Security Onion 2.4.110

### DISCUSSION THREAD

**Discussion by Chizan-Dolrocks** [Security-Onion-Solutions/securityonion#13977] [Unsupported Versions] [answered]
Hello everyone,  

I’m currently using **Security Onion 2.4.110**, and I’ve encountered a couple of issues while working with Sigma detection rules and playbooks. I’d appreciate your insights or guidance.  

1. **Sigma Detection Issue:**  
   - I’ve activated Sigma detection rules, and the corresponding datasource is available in Kibana.  
   - Despite this, I’m not seeing any alerts generated, even though the playbook does detect some activity.  

   **Question:** Is there an additional configuration or tuning required after enabling detection rules (e.g., mapping, thresholds, or other settings)?  

2. **Playbook Customization:**  
   - I’d like to modify playbooks for better prioritization or customization. For instance, I want to create a playbook where detections escalate from “Low” to “High.”  

   **Question:** How can I modify existing playbooks or create new ones to implement such a flow?  

Any suggestions, documentation links, or best practices are highly welcome!  

Thank you in advance for your help!  

Best regards,  
Chireh MOHAMED

**Comment by defensivedepth** [comment]
Hello there - can you confirm what version you are on? Security Onion Playbook was deprecated  & removed prior to 2.4.110.

**Reply by Chizan-Dolrocks** [reply]
currently it is at version 2.4.110

![image](https://github.com/user-attachments/assets/d6c957e7-d824-4ba7-afe6-ff2138680a05)

the details of the playbook is as follows : 

<img width="505" alt="image" src="https://github.com/user-attachments/assets/7c80ce11-4bdb-43ab-adf4-7ba7cdf7448a">

N.B: The link is inaccessible 

![image](https://github.com/user-attachments/assets/797b7662-e85c-4239-85ef-f6f965e8999b)

**Reply by Chizan-Dolrocks** [reply]
Hello, I still had the issue, how can I resolve it?

**Comment by Chizan-Dolrocks** [comment, marked as answer]
I got my answer and it was a problem related to the condition (the selection).
